simple_parsing/wrappers/field_wrapper.py

```diff
@@ -66,6 +79,9 @@ class FieldWrapper(Wrapper[dataclasses.Field]):
     # Whether to add the `dest` to the list of option strings.
     add_dest_to_option_strings: ClassVar[bool] = True
 
+    # Whether to force all options to use its full path
+    force_full_path: ClassVar[FullPathMode] = FullPathMode.DISABLED
```

Comment:
This is more of a choice between 3 values than a boolean, so something like `nested_options_mode` or similar would be a preferable name IMO.